scripts/scripts.js

```diff
@@ -192,7 +193,16 @@ export function addPhotoCredit(pictures) {
  * @return {string|null}
  */
 function findTemplate(className) {
-  return Object.values(ARTICLE_TEMPLATES).find((template) => template === className);
+  return Object.values(ARTICLE_TEMPLATES).find((template) => {
+    if (!className.startsWith(template)) {
```

Comment:
Instead of startsWith I'd prefer to keep exact matching to support templates that start the same eg Gallery and GalleryListicle. You can simply keep the first part of the string before the "(" eg `className.split('(')[0].trim()` to compare with the template.

Reply:
> I could change buildTemplate function to use the metadata instead of the body classlist if you prefer

Yes I think that would make sense